src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/jpa/repository/LduRepository.kt

```diff
@@ -0,0 +1,6 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.integration.jpa.repository
```

Comment:
note: in test resource.

needed to setup ldu in test, can follow the same approach for other WMT data setup rather than https://github.com/ministryofjustice/hmpps-workload/blob/main/src/test/resources/db/data/V2_0__ADD_TEST_DATA.sql

Reply:
would be great if we could get rid of that file, it is confusing